concepts/05 Widgets/Menu/03 Access the Clicked Item.md

```diff
@@ -41,16 +39,69 @@ To access the clicked item, handle the [itemClick](/api-reference/10%20UI%20Widg
         // ...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxMenu ...
+            @item-click="itemClick"
+        />
+    </template>
+    <script>
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import DxMenu from 'devextreme-vue/menu';
+
+    export default {
+        components: {
+            DxMenu
+        },
+        methods: {
+            itemClick(e) {
+                const { itemData, itemElement, itemIndex } = e;
```

Comment:
Do we assign all the [function parameter's fields](https://js.devexpress.com/Documentation/ApiReference/UI_Widgets/dxMenu/Configuration/#onItemClick) to these consts?

Reply:
In this example doesn't have to